cmd/ax/main.go

```diff
@@ -23,8 +23,9 @@ var (
 	alertCommand    = kingpin.Command("alert", "Be alerted when logs match a query")
 	alertDCommand   = kingpin.Command("alertd", "Be alerted when logs match a query")
 	versionCommand  = kingpin.Command("version", "Show the ax version")
+	upgrade         = kingpin.Command("upgrade", "Verify if there is a new version available")
 	addAlertCommand = alertCommand.Command("add", "Add new alert")
-	version         = "dev"
+	version         = "v0.3.0"
```

Comment:
So... do we change `version` manually after releases? Seems fiddly. cc @zefhemel

Reply:
That doesn't seem like a good idea

Reply:
We are replacing the old binary with the new one, so I couldn't do that programmatically, it wont be saved anyway.

Reply:
Reverted to  "dev"